moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -87,6 +98,11 @@ servo:
       default_value: 0.5,
       description: "Max joint angular/linear velocity. Only used for joint commands on joint_command_in_topic."
     }
+    joint_acceleration: {
+      type: double,
+      default_value: 1.0,
+      description: "Scale the maximum allowable joint acceleration specified in the joint limits configuration."
```

Comment:
Can you specify the units here? Assuming `rad/s^2`?

Also if you compare with the other scale parameters, notice those define a maximum velocity whereas this one defines a scale? Make sure it's consistent.

Reply:
I removed all uses of the `joint_acceleration_limit` in favor of taking values from the `robot_state`